code/game/objects/items/stacks/stack.dm

```diff
@@ -31,6 +31,8 @@
 	var/table_type
 	/// If this stack has a dynamic icon_state based on amount / max_amount
 	var/dynamic_icon_state = FALSE
+	/// if true, then this item can stack with children
+	var/parent_stack = FALSE
```

Comment:
do you mean subtypes?

Reply:
Yep, thats what I ment, updating that comment